### DISCUSSION TITLE
Learning videos are gone

### DISCUSSION THREAD

**Discussion by petko** [wprig/wprig#822] [General]
@mor10  Just few weeks ago I was watching the [learning course](https://wprig.io/course/wprig_en_v2/) at wprig.io and they convinced me to use Wprig for my new project.

Now I wanted to see a [specific video](https://wprig.io/learn/load-modular-style-sheets-v2/)  about some Wprig functionality and the video was gone. The whole course seems to be [archived now](https://www.linkedin.com/learning/wordpress-building-progressive-themes-with-wp-rig-2?replacementOf=wordpress-building-progressive-themes-with-wp-rig-2018).

I think this is a big loss for the Wprig community and future users and hope that it can be restored somehow.. Or is there another way to watch these videos?

**Comment by robruiz** [comment]
We have a YouTube account. Just search WP Rig. Also, you can see all of the LinkedIn learning videos here:
https://wprig.io/course/wprig_en_v2/

**Reply by petko** [reply]
Thanks for the YouTube suggestion, I will check it out.

As for the LinkedIn learning videos, you can click on any video page in the link you shared and will see that the actual video is not there, only the text.
